content/smoketest.md

```diff
@@ -462,4 +462,4 @@ The `image-width` shortcode adds an image with alternate text and restricts the
 
 `image-width` example output:
 
-{{< image-width "/images/welcome-guide/ui-editor-labeled.png" "700" "An annotated image of O3DE editor's user interface." >}}
+{{< image-width "/images/welcome-guide/ui-editor-labeled.png" "700" "An annotated image of O3DE editor's user interface." >}}
```

Comment:
This "change" was created accidentally, so it appears in the list of files. It does not impact the PR or make any changes to the file.